### ISSUE TITLE
Correlations do not work

### ISSUE THREAD

**Issue by kdubovikov** [man-group/dtale#578]
An attempt to calculate correlations on any dataset ends with an error:

```
Traceback (most recent call last):
  File "/Users/kdubovikov/miniconda3/envs/ds38/lib/python3.8/site-packages/dtale/views.py", line 113, in _handle_exceptions
    return func(*args, **kwargs)
  File "/Users/kdubovikov/miniconda3/envs/ds38/lib/python3.8/site-packages/dtale/views.py", line 2551, in get_correlations
    matrix_code.format(
KeyError: 'dataset'
```

Tested on Linux, OSX on COVID and Simpsons datasets

**Comment by aschonfeld** [comment]
@kdubovikov great catch! Fix will be in the next release.  Will let you know when that is available.

**Comment by aschonfeld** [comment]
@kdubovikov just released v1.58.1 with a fix for this issue.  If you haven't already, please take a second to toss your ⭐ on the repo.  Thanks 🙏
